src/components/Fetch/index.js

```diff
@@ -291,22 +293,24 @@ componentWillReceiveProps(nextProps){
     if(this.findBookmark(bookmarks, line.lineNumber) !== -1) {
       return true;
     }
-    if(!filter && !inverseFilter) {
+
+    if(filter.length === 0 && inverseFilter.length === 0) {
```

Comment:
You removed a check for filter being null/undefined. I would keep that in unless you have a way to guarantee that they are not